docker/cassandra-diff/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM maven:3-openjdk-11 as builder
+
+# cassandra-diff doesn't have an official Apache distribution yet;
+# we have to build it from the source, here we pin it to a known commit hash
+ENV GIT_ID="4c9bc4f4e3fd7d23b1284c89266ffbf10b8f0183"
+
+RUN git clone https://github.com/apache/cassandra-diff.git cassandra-diff; \
+	cd cassandra-diff; \
+	git checkout $GIT_ID
+
+WORKDIR /cassandra-diff
+RUN mvn clean package -DskipTests
+
+FROM adelphi-dev/spark:3.0.0

```

Comment:
I'm pretty sure this needs to be:

```
FROM spark:3.0.0
```

instead.  I'm assuming the intent here was to reference the version of the Spark image that's built by build-images.sh just before we try to build this guy, but that's just saved as spark:3.0.0.

Reply:
Yup, sure looks like it.  If I try to build as this sits on a newly-initialized OpenStack node:

```
Step 6/7 : FROM adelphi-dev/spark:3.0.0
pull access denied for adelphi-dev/spark, repository does not exist or may require 'docker login': denied: requested access to the resource is denied
```

Just changing to "spark" directly resolves the issue.

Reply:
Is this one we actually want to come from the adelphitools org we created?

I don't think we've published a spark image though.

Reply:
Nah, as you mentioned @jdoneine there isn't a Spark image up there.  We build the Spark image locally in build-images.sh so we actually want the build of cassandra-diff to use _that_ image... which is why I argued for "spark:3.0.0" (i.e. "use my local build of the Spark 3 image").

Reply:
Will address this in small follow-up PR

Reply:
Ah, gotcha.  Sorry, this is the building workflow itself.  Ignore me and move along 😄